lib/src/metta/runner/stdlib.rs

```diff
@@ -147,6 +147,63 @@ fn strip_quotes(src: &str) -> &str {
     src
 }
 
+#[derive(Clone, Debug)]
+pub struct IncludeOp {
+    //TODO-HACK: This is a terrible horrible ugly hack that should be fixed ASAP
+    context: std::sync::Arc<std::sync::Mutex<Vec<std::sync::Arc<std::sync::Mutex<&'static mut RunContext<'static, 'static, 'static>>>>>>,
+}
+
+impl PartialEq for IncludeOp {
+    fn eq(&self, _other: &Self) -> bool { true }
+}
+
+impl IncludeOp {
+    pub fn new(metta: Metta) -> Self {
+        Self{ context: metta.0.context.clone() }
+    }
+}
+
+impl Display for IncludeOp {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "include")
+    }
+}
+
+impl Grounded for IncludeOp {
+    fn type_(&self) -> Atom {
+        Atom::expr([ARROW_SYMBOL, ATOM_TYPE_ATOM, UNIT_TYPE()])
+    }
+
+    fn execute(&self, args: &[Atom]) -> Result<Vec<Atom>, ExecError> {
+        let arg_error = || ExecError::from("include expects a module name argument");
+        let mod_name_atom = args.get(0).ok_or_else(arg_error)?;
+
+        // TODO: replace Symbol by grounded String?
+        let mod_name = match mod_name_atom {
+            Atom::Symbol(mod_name) => mod_name.name(),
+            _ => return Err(arg_error())
+        };
+        let mod_name = strip_quotes(mod_name);
```

Comment:
I am not sure why do we need strip quotes here. Symbol name should not contain quotes AFAIK. Is it needed for some specific usecase?

Reply:
> I am not sure why do we need strip quotes here.

I was following the logic of `import!` and `mod-space!` and didn't give it much thought this time.  But a couple weeks ago there was a bug caused by people (Adam ;-) quoting module names.

It's not necessary to quote module names / mod name paths because spaces are illegal there, but people have quotes in their code as a leftover from when they were file paths, and also when we transition to grounded strings, quotes will be ok again.

So my feeling is to leave this as it is.